src/tracing/CMakeLists.txt

```diff
@@ -41,6 +41,8 @@ add_tracing_library(osd_tp "${osd_traces}" 1.0.0)
 add_tracing_library(rados_tp librados.tp 2.0.0)
 add_tracing_library(rbd_tp librbd.tp 1.0.0)
 add_tracing_library(os_tp objectstore.tp 1.0.0)
+add_tracing_library(functrace_tp functrace.tp 1.0.0)
+add_tracing_library(oidtrace_tp oidtrace.tp 1.0.0)
 
-install(TARGETS rados_tp osd_tp rbd_tp os_tp DESTINATION ${CMAKE_INSTALL_LIBDIR})
+install(TARGETS rados_tp osd_tp rbd_tp os_tp functrace_tp oidtrace_tp DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
this means that `install` will be the only target that depends on `functrace_tp` and `oidtrace_tp`, so there's no guarantee that they will be processed before the targets that actually depend on them. @dlambrig ran into build failures because of this, and had to `$ make functrace_tp oidtrace_tp` before continuing

dependencies for the other _tp targets were added manually:
```
src/librbd/CMakeLists.txt:  add_dependencies(rbd_internal librbd-tp)
src/os/CMakeLists.txt:  add_dependencies(os objectstore-tp)
src/osd/CMakeLists.txt:  add_dependencies(osd oprequest-tp osd-tp pg-tp)
```
we'll need to do the same with these